vendor/github.com/elastic/beats/dev-tools/packer/docker/xgo-image-deb6/beats-builder/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN \
 	rm libpcap0.8-dev*.deb
 RUN \
 	apt-get -o Acquire::Check-Valid-Until=false update && \
-	apt-get install -y libpcap0.8-dev
+	apt-get install -y --allow-unauthenticated libpcap0.8-dev
```

Comment:
You are making changes to the 'vendored' dependencies here. It would be better if the vendored dependencies could remain identical to the versions tagged in `glide.lock`.

If the dependencies need to be changed because they are outdated, would it be possible to use the 'glide' tool to bring them up-to-date, keeping them consistent with upstream?

Reply:
ok. In `glide.lock`, the version for vendor library is a hash of a version or commit Id. I couldn't get the exact string or hash which is inside the file currently. How can i get the value of the version which i want to change and in this case i want to bump to version 5.6.9.